lib/markdownlint-provider.coffee

```diff
@@ -15,10 +18,19 @@ module.exports =
 
       bufferText = textEditor.getText()
 
+      filePath = textEditor.getPath()
+
       options = {
         "strings": { "string": bufferText }
       }
 
+      configPath = find filePath, '.markdownlintrc'
+
+      if configPath
+        configText = fs.readFileSync(configPath, 'utf8')
+        options['config'] = rc.parse(configText)
```

Comment:
Also the atom-linter find function is super nice for finding the rc file at any level, which is the same expected behavior as the cli.

I was originally playing with just using the whole cli to execute the reading of the configuration file and returning the errors, but it became more trouble than it was worth.